server/src/main/java/org/elasticsearch/action/search/TransportSearchAction.java

```diff
@@ -212,18 +283,27 @@ protected void doExecute(Task task, SearchRequest searchRequest, ActionListener<
                 // situations when source is rewritten to null due to a bug
                 searchRequest.source(source);
             }
-            final ClusterState clusterState = clusterService.state();
-            final Map<String, OriginalIndices> remoteClusterIndices = remoteClusterService.groupIndices(searchRequest.indicesOptions(),
-                searchRequest.indices());
+            final SearchContextId searchContext;
+            final Map<String, OriginalIndices> remoteClusterIndices;
+            if (searchRequest.pointInTimeBuilder() != null) {
```

Comment:
Is this condition still required, given `RestSearchAction#preparePointInTime`?

Reply:
We still need this condition. We can execute a search with or without the "pit" parameter.